src/Model/LogEntry.php

```diff
@@ -43,4 +50,93 @@ public function canEdit($member = false, $context = [])
     {
         return false;
     }
+
+    /**
+     * @inheritdoc
+     * @return \SilverStripe\Forms\FieldList
+     */
+    public function getCMSFields()
+    {
+        $config = self::config()->get('format_entry');
+        $fields = parent::getCMSFields();
+        if ($config) { // Default setting, format the entry
+            $fields->removeByName(array('Entry'));
+            $text = $this->Entry;
+            if (!Convert::json2array($text)) { /** @deprecated We're falling back to the legacy text-only error */
+                $entry = $this->convertLegacy($text);
+            } else {
+                $asArray = Convert::json2array($text);
+                $entry = $this->entryToUl($asArray, true);
+            }
+            $fields->addFieldToTab('Root.Main', LiteralField::create('FormattedEntry', nl2br("<pre>" . $entry . "</pre>")));
```

Comment:
Then, you are very free to do that yourself. I refuse to make code less readable by sticking to compliance.